pkg/kudoctl/util/kudo/kudo.go

```diff
@@ -173,16 +174,25 @@ func (c *Client) GetOperatorVersion(name, namespace string) (*v1beta1.OperatorVe
 }
 
 // UpdateInstance updates operatorversion on instance
-func (c *Client) UpdateInstance(instanceName, namespace string, operatorVersionName *string, parameters map[string]string) error {
+func (c *Client) UpdateInstance(instanceName, namespace string, operatorVersion *string, parameters map[string]string, triggeredPlan *string) error {
 	instanceSpec := v1beta1.InstanceSpec{}
-	if operatorVersionName != nil {
+	// 1. new OperatorVersion
+	if operatorVersion != nil {
 		instanceSpec.OperatorVersion = v1core.ObjectReference{
-			Name: kudo.StringValue(operatorVersionName),
+			Name: kudo.StringValue(operatorVersion),
 		}
 	}
+	// 2. new/updated parameters
 	if parameters != nil {
 		instanceSpec.Parameters = parameters
 	}
+	// 3. new/updated execution plan
+	if triggeredPlan != nil {
```

Comment:
Should we validate here if the triggeredPlan is actually a valid plan on the OV?

Reply:
Webhook already has this check so 🤷‍♂